ignore-words.txt

```diff
@@ -0,0 +1 @@
+dragable
```

Comment:
can we make that file private? `.ignore-words.txt`

Reply:
Will do